clusterInfo.py

```diff
@@ -29,33 +29,29 @@ def read_sheet() -> list[list[str]]:
     file = gspread.authorize(credentials)
     sheet = file.open("ANL lab HW enablement clusters and connections")
     sheet = sheet.sheet1
-
-    return [list(e.values()) for e in sheet.get_all_records()]
+    return [{k: str(v) for k, v in record.items()} for record in sheet.get_all_records()]
 
 
 def load_all_cluster_info() -> dict[str, ClusterInfo]:
     cluster = None
     ret = []
     logger.info("loading cluster information")
-    for e in read_sheet():
-        if e[0].startswith("Cluster"):
+    for row in read_sheet():
+        if row[""].startswith("Cluster"):
```

Comment:
Is this relying on the string in Row 1 Column A? Let's update this to something more descriptive

Reply:
Yes its the first column that has an empty column name, would the change be both on the sheet and the code?

Reply:
Yes, updated the spreadsheet, please update in this PR

Reply:
Updated in the PR as well!